src/extensions/scratch3_music/index.js

```diff
@@ -608,6 +608,62 @@ class Scratch3MusicBlocks {
         ];
     }
 
+    /**
+     * An array that is a mapping from MIDI drum numbers in range (35..81) to Scratch drum numbers.
+     * @type {Array[Array]} an array of information about the drums, in the format [drumNum, pitch, decay].
```

Comment:
Decay data is included, but there isn't currently a decay effect in scratch-audio, so I can't do anything with it.